src/calendar-list/index.tsx

```diff
@@ -216,13 +216,16 @@ const CalendarList = (props: CalendarListProps, ref: any) => {
     return false;
   }, [currentMonth]);
 
-  const renderItem = useCallback(({item}: any) => {
+  const renderItem = useCallback(({item}: {item: XDate}) => {
+    const dateString = toMarkingFormat(item);
+    const [year, month] = dateString.split('-');
+    const _testID = `${testID}.item_${year}-${month}`;
```

Comment:
const in a function scope are not supposed to be marked with an underscore... you can simply name it `testId` instead.

Reply:
> const in a function scope are not supposed to be marked with an underscore
Why? 

Do you prefer to have a const that called `testId` and not `_testID`?
The former is more confusing IMO, moreover we usually use `_` prefix for internal variables with a similar name to a prop/argument, it's more explicit and easy to spot.

Reply:
Yes, I prefer `testId`. 
The prefix `_` is used for private (class) variables, inside the function they are all private, so it's wired...

Reply:
I will change. 
just FYI, we used it all the time for function components so it's not that different

Reply:
Yes, we do, in the Component scope (which can be a function, for sure...)